docs/source/environments/braintumor-1d.rst

```diff
@@ -0,0 +1,269 @@
+
+Glioblastomas are fast-growing and highly aggressive, with a median survival time of only 15 months following diagnosis [3]_. Currently the standard treatment approach involves a combination of surgical resection (when possible), external beam radiation therapy (XRT), and concurrent chemotherapy. Magnetic Resonance Imaging (MRI) is used throughout treatment to monitor tumor progression and response to therapy.
+
+Despite known differences in tumor growth across patients, clinical radiation therapy regimens remain largely standardized. This “one-size-fits-all” approach limits the ability to tailor treatment to individual tumors. Computational models such as the DPR framework aim to simulate tumor evolution under different protocols, offering a pathway toward more personalized therapy strategies.
+
+
+
+Dimensional Model
+------------------------
+
+The dimensional model describes tumor growth as occuring in three spatial dimensions and time.
+
+Let :math:`\mathcal{B} = [0, L]` denote the one-dimensional spatial domain of the brain, where :math:`x=0` corresponds to the tumor center and :math:`x=L` is the outer boundary of the simulation domain. This domain models radial tumor expansion under the assumption of spherical symmetry. 
+
+Let :math:`\mathcal{T} = [0, T]` denote the time domain of the simulation. Let :math:`\mathcal{T}_{\text{therapy}}` represent the time intervals at which radiation therapy is administered. The tumor concentration :math:`c(x, t)` evolves according to the following PDE:
+
+.. math::
+
+   \begin{aligned}
+   c_t(x, t) &= \overbrace{D\nabla^2 c}^{\text{diffusion}} + \overbrace{\rho c (1 - c/K)}^{\text{proliferation}} - \overbrace{R(\alpha, \beta, d(x, t), t) c (1 - c/K)}^{\text{radiation}}\,,  \quad && (x, t) \in \mathcal{B} \times \mathcal{T} \\
+   c(x, 0) &= 0.8 K e^{-0.25x^2}, \quad && x \in \mathcal{B}  \\
+   \mathbf{n} \cdot \nabla c(x, t) &=  0\,,  \quad &&  (x, t) \in \partial \mathcal{B} \times \mathcal{T}
+   \end{aligned}
+
+where a zero-flux (Neumann) boundary condition is imposed at the spatial boundaries to ensure that no tumor cells enter or leave the domain during the simulation. :math:`\nabla c` is the spatial gradient of the tumor concentration and :math:`\mathbf{n}` is the outward-pointing unit normal vector at the boundary :math:`\partial \mathcal{B}` of the spatial domain.
+
+The radiation term :math:`R` is given by:
+
+.. math::
+   R(\alpha, \beta,  d(x, t), t) =
+   \begin{cases}
+   1 - S(\alpha, \beta, d(x, t)), & \text{if } t \in \mathcal{T}_{\text{therapy}} \\ 
+   0, & \text{otherwise}
+   \end{cases}
+
+.. math::
+  \begin{aligned}
+   S(\alpha, \beta, d(x, t)) &= e^{-\alpha \cdot \gamma(\alpha, \beta, d(x, t))} \\
+   \gamma(\alpha, \beta, d(x, t)) &= \varphi \cdot d(x,t) \cdot \left( 1 + \frac{d(x, t)}{\alpha / \beta} \right)
+  \end{aligned}
+
+
+**Definitions:**
```

Comment:
instead of writing definitions just like this in bold, you can say
"where we provide the physical interpretation of all the environment variables below:"